### ISSUE TITLE
Polish website

### ISSUE THREAD

**Issue by seanmonstar** [hyperium/hyper#3068]
As part of the 1.0 launch, we should polish https://hyper.rs. The repo to do so is in https://github.com/hyperium/hyperium.github.io.

This is a list of things I came up with, many can likely be broken out into separate issues so they are smaller chunks of work.

- home page
	- update with charter
	- double-block of a summary
	- 6 blocks for tenets
	- user logos
	- last blog post
- navigation
	- guides
	- examples
	- dev/contrib
	- blog
	- icons on right?
		- github
		- discord
		- twitter
	- fix collapse on mobile
- guides
	- update existing guides to hyper 1.0
	- add new hyper-specific guides?
	- add tower section to guides
- examples: pull in hyper repo `examples` directory
- contrib: pull in hyper repo `docs` directory
- blog
	- add blog index page
	- add blog list on left (when viewing post)

**Comment by luooooob** [comment]
Would you like to make a website like tokio.rs?
Obviously, if you want to implement these features, jekyll, mdbook and other tools are not flexible enough, while the front-end technology stack is more convenient to customize the theme or add more features. If you are interested I can help with this

**Comment by seanmonstar** [comment]
No, I don't think there is need to change the stack, at least yet.
